source/dem/pp_contact_force.cc

```diff
@@ -128,30 +128,30 @@ PPContactForce<dim>::update_contact_information(
 template <int dim>
 void
 PPContactForce<dim>::apply_force_and_torque_real(
```

Comment:
Have you considered moving this function to the .h and inlining it? It might be pretty efficient.

Reply:
done.